# 客户端请求时发送的两个数据
int32 num1
int32 num2

# 服务通信中数据分为两部分：请求与响应，在srv文件中使用---将两者分隔开
---

# 服务器相应时发送的一个数据
int32 sum